docker_config/compose_files/.docker/canary.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Do we need docker_config files? Or should we remove them?

Reply:
Just removed them. Sorry about that!